src/schema/rules/checks/fmap.yaml

```
@@ -49,3 +49,16 @@ MagnitudeFileWithTooManyDimensions:
     - nifti_header != null
   checks:
     - nifti_header.dim[0] == 3
+
+EPISmallBVals:
+  issue:
+    code: EPI_WITH_BVALS_NEEDS_SMALL_BVALS
+    message: |
+      EPI field maps with b-values should include small (b<100) values.
+    level: error
+  selectors:
+    - datatype == "fmap"
+    - suffix == "epi"
+    - '"bval" in associations'
+  checks:
+    - min(associations.bval.rows[0]) > 100
```

Note that checks fail if `false`, not `true`.

```suggestion
    - min(associations.bval.values) < 100
```

We also need to add some way to access the bvals to the context:

https://github.com/bids-standard/bids-specification/blob/01025da63297468a3801f8fe43f97a140f2f9551/src/schema/meta/context.yaml#L173-L185

I would recommend

```yaml
            values:
              description: 'B-values contained in .bval file'
              type: array
              items:
                type: number

```

---

And would you mind bumping the SCHEMA_VERSION minor? NBD if not.

---

I totally forgot about that. Thanks!